local/config.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
```

Comment:
@mikaell did you remove this file? There should not be any `local` folder under version control. This should only be just local

Reply:
Yes, it is removed (commit 4a03bb8a).

Reply:
Yes sorry about that, I accidentally looked at an old version